packages/botonic-plugin-hubtype-analytics/src/event-models/ht-event-bot-ai-model.ts

```diff
@@ -0,0 +1,15 @@
+import { EventBotAiModel, RequestData } from '../types'
+import { HtEvent } from './ht-event'
+
+export class HtEventBotAiModel extends HtEvent {
+  intent: string
+  confidence: number
+  confidence_successful: boolean
```

Comment:
why using snake case on the class properties?

Reply:
all names with snake_case is because the tracking endpoint expects them that way